data/drak/drak missions.txt

```diff
@@ -11,17 +11,22 @@
 mission "Drak guarding Korath space"
 	invisible
 	landing
-	npc kill
+	repeat
+	to offer
+		not "Drak guarding Korath space: active"
+	to complete
+		never
+	npc save
 		government Drak
 		personality heroic vindictive staying frugal forbearing
 		system Fasitopfar
 		ship "Archon" "Kasiliri's Folly"
-	npc kill
+	npc save
 		government Drak
 		personality heroic vindictive staying frugal forbearing
 		system Kasikfar
 		ship "Archon B" "Sword of Eden"
-	npc kill
+	npc save
```

Comment:
This makes it so that all three must be killed in order for them to all respawn at once. Should we break these off into their own individual missions so that they can individually respawn?